estmator_project/est_api/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import include, url
+
+urlpatterns = [
+    url(r'^client/', include('est_client.urls')),
+    # url(r'^profile/', include('est_profile.urls')),
+    url(r'^quote/', include('est_quote.urls')),
+]
```

Comment:
I think you probably wanted this `urls.py` file to be in your site folder, not in an app like `est_api`.  This includes urls from the client and the quote apps, which it should not.